.github/workflows/ik-ci.yml

```diff
@@ -0,0 +1,45 @@
+name: InstrumentKit CI
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
I had some issues with another project with github actions, namely that it also runs on forks. Now the `{{ secrets.github_token }}` won't be in that fork, and so the test will fail every time. You can limit that to only run if you are the owner, something like:

``` yaml
jobs:
  test:
    if: github.repository_owner == 'Galvant'
    runs-on: ubuntu-latest
```

Solution is taken from [this discussion](https://github.community/t/have-github-action-only-run-on-master-repo-and-not-on-forks/140840/18). Not sure if it's the most elegant one, but this way actions don't run and fail due to missing secrets on forks.

Reply:
Oh interesting. That's a good idea for the coveralls stage.